### ISSUE TITLE
Sample script

### ISSUE THREAD

**Issue by macMikey** [AndyPiddock/Seth#9]
When I was making the theme I just shared, I was doing most of it without the benefit of having something to look at to make sure it looked the way I wanted it to.  It would be nice to have a sample script to open that has all the various elements in it so as one is fiddling with the theme one can see it live.

**Comment by AndyPiddock** [comment]
Good idea. I'm doing this for the theme exchange so once I have a good test script I'll incorporate it into Seth.
